doc/source/user_guide/session/launching_ansys_fluent.rst

```diff
@@ -300,6 +300,56 @@ can be launched explicitly using :func:`create_launcher() <ansys.fluent.core.lau
   >>> pim_solver_session = pim_solver_launcher()
 
 
+Launching a Fluent Session in Container Mode Using Docker and Podman Compose
+----------------------------------------------------------------------------
+
+You can easily launch an Ansys Fluent session in container mode by leveraging Docker and Podman Compose with PyFluent. 
+This approach enables you to manage multi-container workflows and ensures a consistent, reproducible environment for your simulations.
+
+1. **Docker compose**
+
+    Prerequisites:
+
+    - `Docker <https://www.docker.com/>`_
+    - `Docker Compose <https://docs.docker.com/compose/>`_
+
+    Example:
+
+    .. code:: python
+
+      >>> import os
+      >>> os.environ["PYFLUENT_LAUNCH_CONTAINER"] = "1"
+      >>> os.environ["PYFLUENT_USE_DOCKER_COMPOSE"] = "1"
+      >>> import ansys.fluent.core as pyfluent
+      >>> from ansys.fluent.core import examples
+      >>> solver = pyfluent.launch_fluent()
+      >>> case_file_name = examples.download_file("mixing_elbow.cas.h5", "pyfluent/mixing_elbow")
+      >>> solver.file.read(file_name=case_file_name, file_type="case")
+      >>> solver.exit()
```

Comment:
Split this up and remove duplication. E.g.:

`Set environment variables to select the container engine:`
```python
      >>> import os
      >>> os.environ["PYFLUENT_LAUNCH_CONTAINER"] = "1"
      >>> os.environ["PYFLUENT_USE_DOCKER_COMPOSE"] = "1" # or os.environ["PYFLUENT_USE_PODMAN_COMPOSE"] = "1"
```
Then launch:
```python
      >>> import ansys.fluent.core as pyfluent
      >>> from ansys.fluent.core import examples
      >>> solver = pyfluent.launch_fluent()
      >>> case_file_name = examples.download_file("mixing_elbow.cas.h5", "pyfluent/mixing_elbow")
      >>> solver.file.read(file_name=case_file_name, file_type="case")
      >>> solver.exit()
```

Reply:
Done. Thank you.